report/report-generator.js

```diff
@@ -37,11 +37,10 @@ class ReportGenerator {
       .replace(/</g, '\\u003c') // replaces opening script tags
       .replace(/\u2028/g, '\\u2028') // replaces line separators ()
       .replace(/\u2029/g, '\\u2029'); // replaces paragraph separators
-    const sanitizedJavascript = htmlReportAssets.REPORT_JAVASCRIPT.replace(/<\//g, '\\u003c/');
```

Comment:
Rollup sanitizes the code for us.

Reply:
Does it?

```bash
cat > test.js <<EOF
export function hax() {
  // </script><h1>lulz, pwned!</h1><!--
  return ''
}
EOF

cat > test.html <<EOF
<!doctype html>
<body>
<h1>hello</h1>
<script>
EOF

npx rollup test.js --format=iife >> test.html
open test.html
```

![image](https://user-images.githubusercontent.com/2301202/125306201-a0a50900-e2f4-11eb-9c4e-e7538cc4df5a.png)

Reply:
I think we covered this below, but rollup doesn't, terser *does*. Is that right?

Reply:
yes, that is correct